ecl/eclcc/eclcc.cpp

```diff
@@ -2569,6 +2598,10 @@ int EclCC::parseCommandLineOptions(int argc, const char* argv[])
             else
                 optMetaLocation.clear();
         }
+        else if (startsWith(arg, "--neverSimplify,"))
+        {
+            appendNeverSimplifyList(arg+16);
```

Comment:
None of the other parameters take this form.  The syntax should be 
    --neverSimplify=....

Reply:
I saw this
    -Wc,xx        Pass option xx to the c++ compiler
    -Wl,xx        Pass option xx to the linker
    -Wa,xx        Passed straight through to c++ compiler
    -Wp,xx        Passed straight through to c++ compiler

I thought the coma delineated a list of options and the "=" a signal value.  Is that incorrect?

Reply:
I not sure why those are implemented that way - I think it is because that is the way that the gcc compiler supports passing options so they are implemented so that -Wa,xx would work if passed to eclcc or gcc.

Reply:
It should be using the same calls as the other code:

   if (iter.matchOption(tempArg, "--neversimplify")
... appendNeverSimplifyList)tempArg))

otherwise it is inconsistent.

Also, it looks inconsistent, but generally other options have been all lower case.